content/doc/book/installing/linux.adoc

```diff
@@ -58,27 +58,25 @@ sudo apt-get update
 sudo apt-get install jenkins
 ----
 
-[NOTE]
-====
-If an error is reported, "`jenkins : Depends: daemon but it is not installable`", add the link:https://help.ubuntu.com/community/Repositories/Ubuntu["universe" apt repository] of community maintained free and open source software for Ubuntu by executing this command after `sudo apt-get update`:
-[source,bash]
-----
-sudo add-apt-repository universe
-----
-====
-
 This package installation will:
 
-* Setup Jenkins as a daemon launched on start. See `/etc/init.d/jenkins` for more details.
+* Setup Jenkins as a daemon launched on start. See `/lib/systemd/system/jenkins.service` for more details.
 * Create a '`jenkins`' user to run this service.
 * Direct console log output to the file `/var/log/jenkins/jenkins.log`. Check this file if you are troubleshooting Jenkins.
-* Populate `/etc/default/jenkins` with configuration parameters for the launch, e.g `JENKINS_HOME`
+* Populate `/lib/systemd/system/jenkins.service` with configuration parameters for the launch, e.g `JENKINS_HOME`
 * Set Jenkins to listen on port 8080. Access this port with your browser to start configuration.
 
 [NOTE]
 ====
-If your `/etc/init.d/jenkins` file fails to start Jenkins, edit the `/etc/default/jenkins` to replace the line
-`----HTTP_PORT=8080----` with `----HTTP_PORT=8081----`
+If Jenkins fails to start because a port is in use,
+add the following to `/etc/systemd/system/jenkins.service.d/override.conf`:
```

Comment:
I had never learned how to edit a systemd configuration file.  Would it be worth adding a suggestion here that they can edit the override.conf file with `system edit jenkins`?
```suggestion
add the following to `/etc/systemd/system/jenkins.service.d/override.conf` (for example, using `systemd edit jenkins`):
```